q2_cutadapt/plugin_setup.py

```diff
@@ -6,18 +6,67 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
-import q2_cutadapt
+import importlib
+
+from qiime2.plugin import Plugin, MetadataCategory, Visualization
+from q2_types.multiplexed_sequences import (
+    MultiplexedSingleEndBarcodeInSequence)
+from q2_types.sample_data import SampleData
+from q2_types.per_sample_sequences import SequencesWithQuality
 
-from qiime2.plugin import Plugin
+import q2_cutadapt
+import q2_cutadapt._demux
+from ._format import CutadaptStatsFmt, CutadaptStatsDirFmt
+from ._type import CutadaptStats
 
 
 plugin = Plugin(
     name='cutadapt',
     version=q2_cutadapt.__version__,
     website='https://github.com/qiime2/q2-cutadapt',
     package='q2_cutadapt',
-    description='This QIIME 2 plugin supports removing adapters, primers, '
-                'and other unwanted sequences from sequence data.',
-    short_description='Plugin for removing unwanted sequences from sequence '
+    description='This QIIME 2 plugin uses cutadapt to work with '
+                'non-biological signal in sequences (e.g. barcodes, primers).',
+    short_description='Plugin for handling non-biological signal in sequence '
```

Comment:
Description is a bit unclear/overly specific: "Plugin for handling non-biological signal in sequence data". What about trimming primers, etc — those would still be biological. Maybe (to condense a sentence from the cutadapt docs), something like "plugin for removing adapter sequences, primers, and other unwanted sequence from sequence data"